docs/build/docs.js

```diff
@@ -49,9 +53,12 @@ module.exports = async () => {
   // creates object for docs
   const styleDocs = await styleDocRunner(docsStyles);
   const iconDocs = await iconDocRunner(docsIcons);
+  const bundles = await getBundles(mappedStylesManifest);
+
   const allDocs = {
     styleDocs,
     iconDocs,
+    bundles,
```

Comment:
Note: This is just giving our docs' Nunjucks compile steps the context of the CSS names